src/utils/dialogs.js

```diff
@@ -297,4 +307,136 @@ ExportDialog.propTypes = {
     title: PropTypes.string.isRequired,
 };
 
-export { DeleteDialog, RenameDialog, ExportDialog };
+/**
+ * Dialog to change the access rights of a study #TODO To be moved in the common-ui repository once it has been created
+ * @param {Boolean} open Is the dialog open ?
+ * @param {EventListener} onClose Event to close the dialog
+ * @param studyName the name of the study to export
+ * @param userId the name of the logged in user
+ * @param {String} title Title of the dialog
+ * @param {String} isPrivate tells if the study is private or not
+ */
+const AccessRightsDialog = ({
```

Comment:
dialogs.js is not dedicated to receive all dialogs of the app. Is was just supposed to get very generic and reusable dialogs like "rename". Could you create a access-rights-dialog.js for that?